### PR TITLE
Add pro tips

### PR DESCRIPTION
**Pull request by heratyian**

### CODE REVIEW

**Review by thierrychau — APPROVED** [review]
Looks great! Thanks

**Comment by bchandhu** [comment]
Perfect!

File: content.md [hunk 1]
```diff
@@ -247,6 +247,103 @@ The coding interview is your opportunity to showcase your problem-solving skills
...
+     2| require 'debug'
+     3| 
+     4| def fibonacci(n)
+=>   5|   debugger # breakpoint
+     6|   return n if n <= 1
+     7| 
+     8|   fibonacci(n-1) + fibonacci(n-2)
+     9| end
+    10| 
+=>#0    Object#fibonacci(n=5) at fibonacci.rb:5
```
Comment (ahuynh3a) [inline content.md:342]: Jose and I am got a little confused by these two lines. Could you explain these a little further? Thank you!

**Review by ahuynh3a — COMMENTED** [review]
Everything else looks great!

**Pull request merged**